components/clarinet-deployments/src/lib.rs

```diff
@@ -339,21 +344,28 @@ pub async fn generate_default_deployment(
     let mut requirements_data = BTreeMap::new();
     let mut requirements_deps = BTreeMap::new();
 
+    let mut repl_settings = manifest.repl_settings.clone();
+    repl_settings.remote_data.enabled = false;
```

Comment:
Might need to find a better way to do it.
But we basically want to fully disable remote_data fetching `clarinet check` and other static analyses.
We basically only want remote_data in the repl context (`clarinet console`, js sdk sessions, etc)